app/src/test/java/org/mozilla/fenix/home/ModeTest.kt

```diff
@@ -70,17 +71,19 @@ class ModeTest {
     fun `get current onboarding mode when signed in`() {
         every { onboarding.userHasBeenOnboarded() } returns false
         every { accountManager.authenticatedAccount() } returns mockk()
+        val onboardingConfig = FxNimbus.features.onboarding.value()
 
-        assertEquals(Mode.Onboarding(OnboardingState.SignedIn), currentMode.getCurrentMode())
+        assertEquals(Mode.Onboarding(OnboardingState.SignedIn, onboardingConfig), currentMode.getCurrentMode())
     }
 
     @Test
     fun `get current onboarding mode when signed out`() {
         every { onboarding.userHasBeenOnboarded() } returns false
         every { accountManager.authenticatedAccount() } returns null
         every { accountManager.shareableAccounts(context) } returns emptyList()
+        val onboardingConfig = FxNimbus.features.onboarding.value()
 
-        assertEquals(Mode.Onboarding(OnboardingState.SignedOutNoAutoSignIn), currentMode.getCurrentMode())
+        assertEquals(Mode.Onboarding(OnboardingState.SignedOutNoAutoSignIn, onboardingConfig), currentMode.getCurrentMode())
```

Comment:
`onboarding` is mocked so these will not match when calling `getCurrentMode`

```suggestion
        assertEquals(Mode.Onboarding(OnboardingState.SignedOutNoAutoSignIn, onboarding.config), currentMode.getCurrentMode())
```